src/Planner/PlannerJoinTree.cpp

```diff
@@ -606,6 +607,93 @@ std::unique_ptr<ExpressionStep> createComputeAliasColumnsStep(
     return alias_column_step;
 }
 
+std::unique_ptr<ExpressionStep> createAddingVirtualColumnsStep(
```

Comment:
Name should reflect connection with _table column

Reply:
I've relocated this logic to the common AddingTableNameVirtualColumnStep.cpp file and add "TableName" to the function's name to elucidate its functionality.